pkg/ethtypes/hexbytes.go

```diff
@@ -62,6 +62,14 @@ func (h HexBytes0xPrefix) MarshalJSON() ([]byte, error) {
 	return []byte(fmt.Sprintf(`"%s"`, h.String())), nil
 }
 
+func (h HexBytes0xPrefix) Truncate(length int) HexBytes0xPrefix {
```

Comment:
I'm not sure what this PR does sorry @SamMayWork 

It seems like this is just a bytes manipulation function - surely that could just be done directly by the calling function?
Why is this a helper that is unique to the `HexBytes0xPrefix` type wrapper for `String()` formatting?